assets/js/modules/tagmanager/components/common/SetupUseSnippetSwitch.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Tag Manager Setup Use Snippet Switch component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { MODULES_TAGMANAGER } from '../../datastore/constants';
+import UseSnippetSwitch from './UseSnippetSwitch';
+const { useSelect } = Data;
+
+export default function SetupUseSnippetSwitch() {
+	const primaryContainerID = useSelect( ( select ) =>
+		select( MODULES_TAGMANAGER ).getPrimaryContainerID()
+	);
+
+	const containerID = useSelect( ( select ) =>
```

Comment:
Can we name this variable `existingTag` or something like that? It reads confusing when named `containerID`, it makes me think this is the container ID from the setting, when it is in fact the existing tag.